### DISCUSSION TITLE
Feature request - External relay for each SMTP

### DISCUSSION THREAD

**Discussion by nkgarg21** [postalserver/postal#2046] [Feature suggestions]
Hello Developers,

Is possible to add external relays for each SMTP created in Postal.

for example  - let say we have 3 SMTP created in postal.

We want to create 3 external relays so that SMTP 1 can send through Relay 1, SMTP 2 can send through Relay 2 and SMTP 3 can send through Relay 3.

As i understand right now all the external relays are used by all the SMTP in postal together.

If this is possible, please let me know. i am willing to pay for it.

Thanks

**Comment by thedoom13** [comment]
Hi @nkgarg21, have you solved this problem yet?
I have the same issue and need help. Would anyone happen to have any ideas?

**Comment by jonathansaurens** [comment]
This would be awesome @thedoom13 @nkgarg21 did you guys solve this?
